hack/builder/Dockerfile.cross-compile

```diff
@@ -0,0 +1,18 @@
+ARG BUILDER_IMAGE
+FROM "${BUILDER_IMAGE}"
+
+RUN dnf install -y epel-release && \
+      dnf install -y --setopt=install_weak_deps=False \
+          gcc-aarch64-linux-gnu \
+          gcc-s390x-linux-gnu && \
+      dnf clean -y all
+
+RUN dnf install -y --setopt=install_weak_deps=False  --installroot /usr/aarch64-linux-gnu/sys-root --forcearch aarch64 --releasever 9 \
+          glibc-devel \
+          glibc-static && \
+      dnf clean -y all
+
+RUN dnf install -y --setopt=install_weak_deps=False  --installroot /usr/s390x-linux-gnu/sys-root --forcearch s390x --releasever 9 \
+          glibc-devel \
+          glibc-static && \
+      dnf clean -y all
```

Comment:
A few comments here.

Indentation is off throughout and needs to be adjusted.

Is it useful to have separate `RUN` commands for these, as opposed to a single one? Each `RUN` creates a new container layer. That's useful if these layers can be cached and re-used, but I'm not sure that's the case here. In particular, calling `dnf clean -y all` three times means that we have do download the repo metadata three times instead of just one.

Finally, and this one is just a nit, but I don't like the very long lines, so I'd prefer

```docker
RUN dnf install -y --setopt=install_weak_deps=False \
                   --installroot /usr/s390x-linux-gnu/sys-root \
                   --forcearch s390x \
                   --releasever 9 \
        glibc-devel \
        glibc-static && \
    dnf clean -y all
```

Reply:
I have fixed the indentation.
You are right, i think i put it into separate `RUN` statement to keep the arches a bit more separated, but not downloading the metadata 3 times makes sense.